docs/django-widget.rst

```diff
@@ -56,9 +56,21 @@ widget url:
 
     UPLOADCARE = {
         # ...
-        'upload_base_url' = 'http://path.to/your/upload/handler',
+        'upload_base_url': 'http://path.to/your/upload/handler',
     }
 
+If you have signed uploads enabled in your Uploadcare project, widget-based uploads will fail unless you enable the ``signed_uploads`` setting in your Django project.
+
+Note that in this case, the benefits of signed uploads are essentially negated for user uploads. So enabling this feature is only feasible if, for example, files are only uploaded via the admin interface.
```

Comment:
Maybe write this note in the changelog too?

Reply:
the note is confusing and I believe incorrect.
public key leak is still a thing.

Reply:
Ok, need to rephrase it. At the moment I have no idea how exactly.